### ISSUE TITLE
Folder structure/workflow with Angular plugins?

### ISSUE THREAD

**Issue by PeterStaev** [NativeScript/nativescript-plugin-seed#161]
I can't find anywhere how should the plugin be structured and what should be the workflow when the plugin as an android code - for example the GridView or DropDown plugins.

**Comment by elena-p** [comment]
You could find more information on how to build angular wrappers [here](https://docs.nativescript.org/angular/plugins/angular-plugin).

However, it would be a good improvement to include the steps in the plugin seed itself.

**Comment by PeterStaev** [comment]
@elena-p , yes, I'm well aware of how wrappers should be built 😉  I'm more talking about to have in the seed a predefined structure where those wrappers should be put. Also I think best would be to have the postclone script automate this: so ask the user if their plugin will have angular wrappers, if Yes then: 
* Add a dummy wrapper in the predefined folder
* Add angular compiler dependencies with their correct versions to the `package.json` file
* Add angular compiler scripts to `packages.json` and include them in the build script
* Automatically add the angular demo w/o prompting the user

Also it would be good if we can return and execute this part in the later stages so in case I start a plugin w/o angular wrappers I can later return and run some scripts that performs the above mentioned steps.

**Comment by edusperoni** [comment]
I'd also like to add that the docs (https://docs.nativescript.org/angular/plugins/angular-plugin) do not mention the angular compiler at all, which results in many plugins not being compatible with AOT, for example: https://github.com/Akylas/nativescript-material-components/pull/36

I've added a similar issue a while back: https://github.com/NativeScript/nativescript-plugin-seed/issues/126
